src/v/serde/parquet/metadata.h

```diff
@@ -329,6 +439,13 @@ struct row_group {
     /** Byte offset from beginning of file to first page (data or dictionary)
      * in this row group **/
     int64_t file_offset;
+
+    /** Total byte size of all compressed (and potentially encrypted) column
+     * data in this row group **/
+    int64_t total_compressed_size;
```

Comment:
this is great, it would be great if this is accessible from `writer` interface to control the size of parquet files

Reply:
Good point - I have only been thinking about memory pressure. Will add a commit for this

Reply:
done